sling/core/commons/src/main/java/com/composum/sling/core/util/RequestUtil.java

```diff
@@ -131,6 +131,15 @@ public static int getParameter(SlingHttpServletRequest request, String name, int
         return result;
     }
 
+    public static boolean getParameter(SlingHttpServletRequest request, String name, boolean defaultValue) {
+        Boolean result = defaultValue;
+        String string = request.getParameter(name);
+        if (StringUtils.isNotBlank(string)) {
+            result = Boolean.parseBoolean(string);
+        }
+        return result != null ? result : defaultValue;
```

Comment:
result can be 'null' if the 'defaultValue' - this i a case used really in the code